lib/editor-socketio-server.js

```diff
@@ -15,14 +15,16 @@ function DummyLogger () {
   };
 }
 
-function EditorSocketIOServer (document, operations, docId, mayWrite, operationCallback, operationEventCallback) {
+function EditorSocketIOServer (document, operations, docId, mayWrite, operationCallback, operationEventCallback, adaptiveAck) {
   EventEmitter.call(this);
   Server.call(this, document, operations);
   this.users = {};
   this.docId = docId;
   this.mayWrite = mayWrite || function (_, cb) { cb(true); };
   this.operationCallback = operationCallback;
   this.operationEventCallback = operationEventCallback;
+  this.adaptiveAck = adaptiveAck || false;
+  this.adaptiveAckDelay = 2000;
```

Comment:
How about put adaptiveActDelay as the function argument? And check whether `adaptiveAckDelay` is number and enable it.

Reply:
sounds ok